parent-pom.xml

```diff
@@ -507,6 +507,12 @@
         <version>${version.plugin.surefire}</version>
         <scope>test</scope>
       </dependency>
+      <dependency>
+        <groupId>org.wiremock</groupId>
+        <artifactId>wiremock-standalone</artifactId>
+        <version>3.8.0</version>
```

Comment:
add version to the properties as for other dependencies

Reply:
done